.github/workflows/documentation.yaml

```diff
@@ -15,6 +15,9 @@ jobs:
         - cloudevent-trigger
         - cloudevent-recorder
         - networking
+        - dashboard/service
+        - dashboard/job
+        - dashboard/topic
```

Comment:
Do we want the tiles too?

Reply:
subscription?

Reply:
Ehhh, maybe. They're so simple I think we'd end up having most of the content of the directories be comments, and generated markdown from comments.

Which maybe isn't a bad thing! I just don't want to have to write it all right now 😆

Reply:
LOL, ok.  I've been trying to keep the bar here relatively high, so let's take the AI to chase this once we feel like the shape is something we're really bought into?

Reply:
re: topic vs subscription, wdyt about `dashboard/subscriptions` (with an `s`)?

The PubSub dashboard is weird because it doesn't just go by the (regionally-unique) name of the thing, it goes by prefixes on (globally-unique) subscription names.

Reply:
The sent-events tile for instance relies on the fact that `subscription_prefix` is also a topic name.

Reply:
We should be able to filter subscriptions metrics on `topic_id` without assuming the subscription prefix matches.  I was playing with this in metrics explorer earlier.

Reply:
But let's land the widgets / cloudrun dashboards, and then talk pubsub next